source/material_model/rheology/visco_plastic.cc

```diff
@@ -261,22 +286,21 @@ namespace aspect
                     Assert(std::isfinite(in.strain_rate[i].norm()),
                            ExcMessage("Invalid strain_rate in the MaterialModelInputs. This is likely because it was "
                                       "not filled by the caller."));
-                    const SymmetricTensor<2,dim> effective_strain_rate =
-                      elastic_rheology.calculate_viscoelastic_strain_rate(in.strain_rate[i],
+
+                    // The second moment invariant is returned.
```

Comment:
This comment at first confused me, nothing is "returned" and the process of computing the strain rate is explained in the comment in line 280. If you decide to keep this comment please move it one line down (where the second moment invariant is computed). Or you can delete it completely.